source/Body.cpp

```diff
@@ -588,6 +589,7 @@ Body::doRHS()
 		vec6 F6_i;
 		mat6 M6_i;
 		attached->getNetForceAndMass(F6_i, M6_i, r7.pos);
+		F6_i(Eigen::seqN(0, 3)) += attached->getCentripetalForce(r7.pos, v6);
```

Comment:
Should this be moved into getNetForceAndMass in the interest of keeping consistent function names? NetForce to me includes this centripetal force, and that is also where forces are translated to be about the body's ref point

Reply:
Fixed in 94b78a27a35b47180730db5f3c8d47eac4726238